src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/CVObjectDetectionKeyFrameCriteriaData.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using JetBrains.Annotations;
+using RegressionGames.StateRecorder.BotSegments.JsonConverters;
+using RegressionGames.StateRecorder.JsonConverters;
+
+namespace RegressionGames.StateRecorder.BotSegments.Models
+{
+    [Serializable]
+    public class CVObjectDetectionKeyFrameCriteriaData : IKeyFrameCriteriaData
+    {
+        // update this if this schema changes
+        public int apiVersion = SdkApiVersion.VERSION_9;
+
+        public string text;
+        /**
+       * base64 encoded byte[] of jpg image data , NOT the raw pixel data, the full jpg file bytes
+       */
+        // public string imageData; This will be added in the next PR.
```

Comment:
If there is a ticket for this, we usually do the following so we know:

```suggestion
        // TODO(REG-???): Supporting image queries will be added soon
        // public string imageData;
```

Reply:
Done